pandas/tests/tools/test_to_datetime.py

```diff
@@ -2976,24 +2972,17 @@ def test_na_to_datetime(nulls_fixture, klass):
         assert result[0] is NaT
 
 
-def test_empty_string_datetime_coerce_format():
-    # GH13044
+@pytest.mark.parametrize("errors", ["raise", "coerce", "ignore"])
+def test_empty_string_datetime_coerce_format(errors):
```

Comment:
Nit: Maybe good to update this test name since `coerce` isn't the only errors argument being tested here